proposals/mftf-package.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+Magento Functional Testing Framework (MFTF) with 2.3.0 release is going to be a require-dev dependency in main magento composer.json. Details described in this document https://wiki.corp.magento.com/display/QMT/Mftf+Composer+Package+and+Decoupled+Structure
```

Comment:
Please list any necessary details here directly. Internal documents are not accessible for the community.